packages/ds-mui-theme/src/components/SearchableSelect.js

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Select from 'react-select';
+import {withStyles} from '@material-ui/core/styles';
+import {MenuItem, ListItemText, TextField, Paper} from '@material-ui/core';
+import Typography from './Typography'
+
+const styles = theme => ({
+    root: {
+        flexGrow: 1
+    },
+    input: {
+        display: 'flex',
+        padding: 0
+    },
+    valueContainer: {
+        display: 'flex',
+        flexWrap: 'wrap',
+        flex: 1,
+        alignItems: 'center',
+        overflow: 'auto'
+    },
+    noOptionsMessage: {
+    },
+    singleValue: {
+        fontSize: theme.typography.delta.fontSize
+    },
+    placeholder: {
+        position: 'absolute',
```

Comment:
I think absolute position must not be in design system component, as it break the html flow